src/main/webapp/js/instructorStudentEdit.js

```diff
@@ -0,0 +1,16 @@
+$(document).ready(function(){
+    readyStudentEditPage();
+});
+
+function readyStudentEditPage(){
+    
+    $("#button_submit").click(function(){
```

Comment:
I have changed that the warning will show only when the team info is edited.

I am thinking if in the future we are going to allow user to download any combination of the statistics, we can add a button that directly link to the download page that all stats is related to the student.